<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="RenderArea" name="renderArea" native="true">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>16</y>
      <width>291</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>140</width>
      <height>300</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnAstroid">
       <property name="text">
        <string>Astroid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCycloid">
       <property name="text">
        <string>Cycloid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHuygens">
       <property name="text">
        <string>Huygens</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHypo">
       <property name="text">
        <string>Hypo Cycloid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLine">
       <property name="text">
        <string>Line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCircle">
       <property name="text">
        <string>Circle</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Scale :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="spinScale">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Interval Length :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="spinInterval">
         <property name="decimals">
          <number>1</number>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Steps :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinSteps">
         <property name="maximum">
          <number>512</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RenderArea</class>
   <extends>QWidget</extends>
   <header location="global">renderarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
